server/app.ts

```diff
@@ -35,10 +36,11 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpWebRequestParsing())
   app.use(setUpStaticResources())
   nunjucksSetup(app, services.applicationInfo)
+  app.use(setupSmokeTest(services.smokeTestStore))
   app.use(setUpAuthentication())
-  app.use(authorisationMiddleware())
   app.use(setUpCsrf())
   app.use(setUpCurrentUser(services))
+  app.use('/', authorisationMiddleware(['ROLE_SLM_SCAN_BARCODE', 'ROLE_SLM_ADMIN']))
```

Comment:
```suggestion
  app.use(authorisationMiddleware(['ROLE_SLM_SCAN_BARCODE', 'ROLE_SLM_ADMIN']))
```
Minor (and fine as it is) but I don't think you need the `'/'` because it's the default. Probably made sense when there were two methods of authentication.
(Same applies to the line below, I imagine).